### PR TITLE
downgrade pytorch to 1.8.0

### PR DESCRIPTION
**Pull request by Renkai**
Related issue: https://github.com/eto-ai/rikai/issues/342
Related pr: https://github.com/eto-ai/rikai/pull/347

Downgrade PyTorch version to align with Databricks platform, but we can still get rid of a customized `Dataloader`, we can upgrade it again when Databricks platform upgrades.

### CODE REVIEW

File: python/setup.py [hunk 1]
```diff
@@ -24,7 +24,7 @@
     "wheel",
 ]
 sklearn = ["scikit-learn"]
-torch = ["torch>=1.9.0", "torchvision>=0.11.0"]
+torch = ["torch>=1.8.0", "torchvision>=0.11.0"]
```
Comment (eddyxu) [inline python/setup.py:27]: do we need to change torchvision version as well?

Comment (da-tubi) [reply python/setup.py:27]: See https://docs.databricks.com/release-notes/runtime/8.3ml.html

Comment (da-tubi) [reply python/setup.py:27]: torchvision 0.9.1

Comment (Renkai) [reply python/setup.py]: updated

**Review by darcy-shen — APPROVED** [review]
LGTM

**Pull request merged**